src/Test/Internal.elm

```diff
@@ -19,6 +19,14 @@ failNow record =
         (\_ _ -> [ Test.Expectation.fail record ])
 
 
+emptyDescriptionFailure : Test
+emptyDescriptionFailure =
+    failNow
+        { description = "This test has an empty description. Let's give it a nonempty one!"
```

Comment:
I agree with the "useful" rephrasing, but I'm not quite sure which work to use for the emptiness. If you refer to it as "blank", then you should also give that same error message on a whitespace-only description; I have no idea if we do that right now.